gensim/models/phrases.py

```diff
@@ -482,19 +479,17 @@ def learn_vocab(sentences, max_vocab_size, delimiter=b'_', progress_per=10000,
                     "PROGRESS: at sentence #%i, processed %i words and %i word types",
                     sentence_no, total_words, len(vocab),
                 )
-            s = [utils.any2utf8(w) for w in sentence]
-            last_uncommon = None
-            in_between = []
-            for word in s:
+            common = []
```

Comment:
Also, `common` isn't a particularly good variable name here, because it clashes with `common_terms` and means something completely different.

I think something like `bigram` would be a better name.